bokehjs/src/less/continuum.less

```diff
@@ -16,7 +16,7 @@
 .tableheader {
   background-color: #C0C0C0;
 }
-#notebook .bk-plot-wrapper table {
```

Comment:
Anyway, removing fixed ID-based styling is mandatory as this is something a **library** must not use.